tests/code-splitting/subtests/splitting-modules/subtests/multi-entry/webpack/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-result: fail
+result: pass
 ---
 
-Webpack extracts the common `objects.js` module as-is into its own shared bundle.
+Webpack optimizes modules separately for each runtime. As each entry has it's own runtime, modules are optimized for each entry.
```

Comment:
Yeah, that's (nearly) best we can do. When multiple entries can be loaded on the same page (with runtimeChunk: "single"), we can't optimize modules of entrypoints independently, since they could contain state, which would be duplicated when modules are duplicated for independent optimization.

Technically it's actually possible, but very difficult to do. We would have to extract the shared state into a separate module and reference that from two independent optimized modules. That would be really tricky and would require to bundle on statement level instead of on module level.